generic/service_check.py.data/services.cfg

```diff
@@ -3,7 +3,7 @@ services = sshd,ufw,networking,libvirtd
 [rhel]
 services = sshd,firewalld,network,libvirtd
 [SuSE]
-services = sshd,SuSEfirewall2,network,libvirtd
+services = sshd,network,libvirtd
```

Comment:
can we just depend on system service using chkconfig not depend on cfg file

Reply:
@PraveenPenguin chkconfig shows only SysV services and does not include distro specific services. Will `systemctl list-unit-files` help here? If so, we may have to make this cfg file obsolete.

Reply:
sure that will help but we need to have support for older distro as well so i think both need to be used

Reply:
@PraveenPenguin Sure, as discussed we can make this approach an enhancement for the test in future.

Reply:
sure , will merge